From cd5b34e18ab1ec438500816e91a56e981d5b990d Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Wed, 31 May 2017 10:44:19 +0800
Subject: [PATCH] arm: dts: actduino_bubble_gum_sdboot_linux: modify
 atc260x-cap-gauge node according to experiment outcome.

According to battery experiment outcome from hardware vendor, modify
atc260x-cap-gauge node, which is the configuration of gauge function.
Now, with this configuration, battery volume can reach 100% and then
stop charging.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/actduino_bubble_gum_sdboot_linux.dts  |   24 ++++++++++----------
 1 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/actduino_bubble_gum_sdboot_linux.dts b/arch/arm/boot/dts/actduino_bubble_gum_sdboot_linux.dts
index d954ba9..e3ccb28 100755
--- a/arch/arm/boot/dts/actduino_bubble_gum_sdboot_linux.dts
+++ b/arch/arm/boot/dts/actduino_bubble_gum_sdboot_linux.dts
@@ -617,25 +617,25 @@
 				capacity = < 5200 >;                             /*mAh, the full capacity of battery used*/
 				icm_available= < 1 >;				  /* 1:available 0:not available */
 				icm_ohm_val = < 20 >;                             /*mohm,just 20 or 10,for ICM detecting current*/
-				taper_voltage = < 4360 >;                         /*mV,the one close to the full capacity*/
+				taper_voltage = < 4320 >;                         /*mV,the one close to the full capacity*/
 				taper_current = <52>;
-				min_over_chg_protect_voltage = < 4410 >;	  /*mA,the one close to the full capacity*/
+				min_over_chg_protect_voltage = < 4395 >;	  /*mA,the one close to the full capacity*/
 				terminal_voltage = <3698>;                        /*mV,the lowest voltage of battery permitted*/
 				suspend_current = < 7730 >; 			  /* uA,the current of suspend*/
 				shutdown_current = < 50 >;                        /* uA,the current of shutdown*/
 				print_switch = <0>;                               /*0:open the switch of printing information 1:the opposite*/
 				log_switch = <0>;                                 /*0:open the switch of recording charging log 1:the opposite*/
 				
-				ocv_soc_00 = <3434 3537 3604 3652 3687 3696 3698 3700 3700 3704>;    	 /* capacity:from 1% to 10%*/
-				ocv_soc_10 = <3709 3714 3722 3729 3737 3744 3750 3755 3760 3766>;	 /* capacity:from 11% to 20%*/
-				ocv_soc_20 = <3772 3775 3777 3780 3783 3787 3790 3793 3796 3797>;	 /* capacity:from 21% to 30%*/
-				ocv_soc_30 = <3800 3803 3805 3807 3810 3813 3816 3819 3823 3826>;	 /* capacity:from 31% to 40%*/
-				ocv_soc_40 = <3830 3834 3838 3842 3847 3852 3857 3861 3867 3873>;	 /* capacity:from 41% to 50%*/
-				ocv_soc_50 = <3880 3886 3893 3901 3911 3923 3933 3942 3951 3957>;	 /* capacity:from 51% to 60%*/
-				ocv_soc_60 = <3966 3873 3981 3987 3996 4005 4012 4020 4029 4039>;	 /* capacity:from 61% to 70%*/
-				ocv_soc_70 = <4049 4059 4068 4076 4087 4097 4106 4116 4126 4136>;	 /* capacity:from 71% to 80%*/
-				ocv_soc_80 = <4147 4156 4168 4178 4192 4198 4209 4222 4100 4233>;	 /* capacity:from 81% to 90%*/
-				ocv_soc_90 = <4244 4255 4267 4278 4289 4301 4312 4325 4336 4350>;	 /* capacity:from 91% to 100%*/
+				ocv_soc_00 = <3429 3532 3600 3647 3684 3694 3698 3699 3700 3704>;    	 /* capacity:from 1% to 10%*/
+				ocv_soc_10 = <3708 3712 3718 3725 3733 3741 3747 3753 3757 3760>;	 /* capacity:from 11% to 20%*/
+				ocv_soc_20 = <3769 3774 3776 3779 3781 3784 3789 3792 3795 3796>;	 /* capacity:from 21% to 30%*/
+				ocv_soc_30 = <3796 3800 3802 3805 3808 3811 3812 3816 3819 3822>;	 /* capacity:from 31% to 40%*/
+				ocv_soc_40 = <3825 3828 3833 3837 3842 3846 3850 3854 3860 3865>;	 /* capacity:from 41% to 50%*/
+				ocv_soc_50 = <3870 3874 3881 3888 3896 3903 3915 3925 3935 3944>;	 /* capacity:from 51% to 60%*/
+				ocv_soc_60 = <3952 3957 3967 3973 3982 3989 3996 4005 4010 4021>;	 /* capacity:from 61% to 70%*/
+				ocv_soc_70 = <4028 4036 4045 4056 4065 4074 4083 4093 4102 4110>;	 /* capacity:from 71% to 80%*/
+				ocv_soc_80 = <4120 4130 4140 4151 4160 4171 4283 4189 4192 4201>;	 /* capacity:from 81% to 90%*/
+				ocv_soc_90 = <4212 4216 4220 4224 4228 4232 4236 4240 4244 4250>;	 /* capacity:from 91% to 100%*/
 			};
 
 			atc260x-backup{
-- 
1.7.5.4

